addon/services/user-activity.js

```diff
@@ -98,6 +103,9 @@ export default Service.extend(Evented, FastBootCompatMixin, {
     if (this.has('userActive')) {
       this.trigger('userActive', event);
     }
+    if(this._eventsListened.indexOf('storage') > -1 && storageAvailable('localStorage')) {
+      localStorage.setItem(this.localStorageKey, new Date());
```

Comment:
Why `new Date()`?

Reply:
The storage event is only triggered when a value is changed, so the easiest way to get a new value was to get a new date. Another way would of course be to read back the current value and updating that but it would require more code with little gain.

Reply:
Can you add a very brief comment on this line to that effect?

Reply:
Sure thing, I've added it.